agent/config/parse_windows_test.go

```diff
@@ -16,10 +16,14 @@
 package config
 
 import (
+	"github.com/golang/mock/gomock"
```

Comment:
nit: to align with convention in other files, please keep 3rd parties import below built in import (i.e. "os" and "testing"). example: https://github.com/aws/amazon-ecs-agent/blob/master/agent/config/config.go#L16-L31